mlir/utils/performance/tuningRunner.py

```diff
@@ -226,7 +226,7 @@ def main(args=None):
     if parsed_args.config:
         configs = parsed_args.config
     elif opType == Operation.CONV:
-        configs = perfRunner.getConvConfigurations(paths.configuration_file_path)
+        configs = perfRunner.getConvConfigurations(paths.configuration_file_path, do_sweep=False)
```

Comment:
I don't think is the right behavior? The resnet50 file is set up to expect a sweep?

Or at least add a flag?

Reply:
flag you mean in the config file ? 
(whether to do a sweep)

I ll describe the problem:
* In the MIGraphX tuning prototype it relies on tuning key to extract exact configs that needs to be tuned.
* Since the tuning key already has layout and friends already populated, they wont work as they work with gemm s.
* In order to not to overtune (i.e. not tune more than what client) asks for -- we might need a switch.

Having said that, did you mean a flag to indicate a sweep in the config line ?
OR I could do a check to see what is already present and do the sweep for rest ?

Reply:
What I'm saying is that `mlir/utils/jenkins/miopen-tests/resnet50-miopen-configs` (the default config file) doesn't indicate fwd/bwd/..., and the convolution tuning flow takes that config and expects that you run a sweep over it.

Reply:
(rather than assuming its not there)

Reply:
Yeah, I think sweeping on unspecified parameters is the right call - I _think_ we do that for GEMM performance tuning, for example.